resources/js/Pages/Company/Project/Tasks/Partials/ProjectTaskCheckbox.vue

```diff
@@ -87,15 +87,15 @@ input[type=checkbox] {
       <div class="flex items-start">
         <input
           :id="id"
-          v-model="updatedValue"
+          v-model="proxyValue"
+          :value="value"
           :data-cy="datacy + '-single-item'"
           type="checkbox"
           class="relative"
           :class="classes"
           :required="required"
           :name="name"
           :disabled="!editable"
-          @change="emitValue()"
```

Comment:
This feature is also broken. We can't assign someone to a task.

Reply:
fixed